### PR TITLE
Adding XLC16 to aix playbook as its a requirement for JDK13

### PR DESCRIPTION
**Pull request by jdekonin**
* renamed existing registers and tags for xlc -> xlc13
* added a cleanup for xlc13 and xlc16 temporary files

Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

**Comment by jdekonin** [comment]
OpenJ9 requirement for JDK13.  Either installing xlc16 or installing the APAR in the openj9 issue install libc++.rte.
ttps://github.com/eclipse/openj9/issues/7579

File: ansible/playbooks/aix.yml [hunk 1]
```diff
@@ -399,78 +399,132 @@
...
+            src: /Vendor_Files/aix/XLC/XL_C_Cpp_FOR_AIX_V16.1_EMG.tar.Z
+            dest: /tmp
+            remote_src: no
+          when: xlc16.stat.islnk is not defined
+          tags: xlc16
+
+        - name: Install IBM XLC16 - installp
+          command: installp -aXYgd /tmp/usr/sys/inst.images all
+          register: result.xlc
+          ignore_errors: yes
```
Comment (vsebe) [inline ansible/playbooks/aix.yml]: Why are errors ignored?

File: ansible/playbooks/aix.yml [hunk 2]
```diff
@@ -399,78 +399,132 @@
...
+          register: xlc16_qversion
+          tags: xlc16
+
+        - name: Display XLC16 qversion information
+          debug:
+            msg: "{{ xlc16_qversion.stdout }}"
+          tags: xlc16
+
+        - name: Clean XLC16 tmp files
+          file:
```
Comment (vsebe) [inline ansible/playbooks/aix.yml]: There is also a log file that needs to be deleted: /tmp/install.log
You could also add `-e /tmp/usr/install.log` to the`installp ...` command above, thus no extra clean up required

File: ansible/playbooks/aix.yml (hunk 1 above)
Comment (jdekonin) [reply ansible/playbooks/aix.yml]: There is often soft requirements missing that cause the installp process to return a non-zero code even though the compiler installs fine.  Example: a non-english language pack for error messages/logging.

File: ansible/playbooks/aix.yml (hunk 2 above)
Comment (jdekonin) [reply ansible/playbooks/aix.yml]: Good idea; updated both cases.  Is this how you mean?

File: ansible/playbooks/aix.yml [hunk 3]
```diff
@@ -465,78 +465,132 @@
...
+        - name: Transfer and Extract XLC16
+          unarchive:
+            src: /Vendor_Files/aix/XLC/XL_C_Cpp_FOR_AIX_V16.1_EMG.tar.Z
+            dest: /tmp
+            remote_src: no
+          when: xlc16.stat.islnk is not defined
+          tags: xlc16
+
+        - name: Install IBM XLC16 - installp
+          command: installp -aXYgd -e /tmp/usr/install.log /tmp/usr/sys/inst.images all
```
Comment (vsebe) [inline ansible/playbooks/aix.yml]: `/tmp/usr/sys/inst.images` is the install images location directory which is specified with `-d` options.
 Move  `/tmp/usr/sys/inst.images` after `-aXYgd`:  e.g. `installp -aXYgd /tmp/usr/sys/inst.images -e /tmp/usr/install.log all` or 
move the `d` option: e.g. `installp -aXY -e /tmp/usr/install.log  -d /tmp/usr/sys/inst.images all`

File: ansible/playbooks/aix.yml (hunk 2 above)
Comment (vsebe) [reply ansible/playbooks/aix.yml]: see https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/773/files#r349738526

File: ansible/playbooks/aix.yml (hunk 3 above)
Comment (jdekonin) [reply ansible/playbooks/aix.yml]: Of course.  PR updated.

**Comment by jdekonin** [comment]
@sej-jackson @sxa555 can I please get a review on this PR?

**Pull request merged**